static/src/javascripts/projects/common/modules/mark-candidates.js

```diff
@@ -1,5 +1,5 @@
 export const markCandidates = (exclusions, winners, options) => {
-	if (!options.debug) return winners;
+	if (!options?.debug) return winners;
```

Comment:
[liveblog-adverts calls fillSpace](https://github.com/guardian/frontend/blob/721529885b15acdd616cfd9586a837b98893a0c8/static/src/javascripts/projects/commercial/modules/liveblog-adverts.js#L92) without passing an options object.

Reply:
Converting `spacefinder` and `mark-candidates` to typescript would catch the fact that `options` is possible undefined as well I think.